test/smalltalk.js

```diff
@@ -412,6 +422,24 @@ test('smalltalk: alert: click', (t) => {
     t.end();
 });
 
+test('smalltalk: alert: custom label', (t) => {
+    before();
+    
+    const el = {
+        innerHTML: ''
+    };
+    
+    const createElement = getCreateElement(el);
+    global.document.createElement = createElement;
+    
+    var options = {'buttons' : {'ok':'Ok'}};
```

Comment:
Could you please change `var` with `const` and write each object property in own line

Reply:
You mean : 
```
var options = {
  'buttons' : {
    'ok':'Ok'
  }
};
```

Thanks

Reply:
Yes, and would be better without quotes